zarf.yaml

```diff
@@ -44,7 +44,38 @@ components:
           - cmd: k3d cluster delete ${ZARF_VAR_CLUSTER_NAME}
             description: "Destroy the cluster"
 
-  - name: create-cluster
+  - name: k3d-airgap-images
+    required: true
+    only:
+      flavor: airgap
+    description: "Load the airgap images for k3d into Docker"
+    import:
+      path: airgap/k3d
+      name: k3d-airgap-images
+    
+  - name: create-cluster-airgap
+    required: true
+    only:
+      flavor: airgap
+    actions:
+      onDeploy:
+        before:
+          - cmd: |
+              echo ""
+              echo "###################################################"
+              echo "#                                                 #"
+              echo "#      PACKAGE BEING DEPLOYED IN AIRGAP MODE      #"
+              echo "#    THIS IS NOT MEANT TO BE USED IN PRODUCTION   #"
+              echo "#                                                 #"
+              echo "###################################################"
+              echo ""
+          - cmd: echo "true"
+            description: "Set AIRGAP_MODE to true"
+            mute: true
+            setVariables:
+              - name: AIRGAP_MODE
+
+  - name: create-cluster-no-airgap
```

Comment:
Is this a poor component name, since it's actually run for either option (airgap or not) - only difference being a volume mount inside of the component/action?